swal-forms.js

```diff
@@ -219,7 +219,8 @@
         return inputTag + labelTag
 
         function toHtmlOptions (optionsString, option) {
-          return optionsString + t("<option value='{value}'>{text}</option>", option)
+            option.selected = ((typeof option.selected !== 'undefined') && option.selected) ? ' selected' : ''
```

Comment:
I rarely write on js, but when i left only `option.selected`, Chrome show the error. So i save this condition in my project. If you sure, it can be  easier